admin/webUI/app/components/invite/views/invite-dashboard.html

```diff
@@ -97,4 +107,24 @@
       </div>
     </div>
   </div>
+  <dialog id="dialog" class="mdl-dialog">
+    <h4 class="mdl-dialog__title">Cancel</h4>
```

Comment:
This Dialog Box won't work. Need to remove this. This dialog modal is from official MDL doc: https://getmdl.io/components/index.html#dialog-section.

The first line says : **"Note: Dialogs use the HTML <dialog> element, which currently has very limited cross-browser support. To ensure support across all modern browsers, please consider using a polyfill or creating your own. There is no polyfill included with MDL."**

So `<dialog>` tag has less support. It is only supported by Chrome 37+  and Opera 24+. So it won't work in Firefox, Safari, IE, Edge, other browsers and versions. 

[Dialog support in browsers](http://caniuse.com/#feat=dialog)